book/030-tutorial-downstream/060-alpha-diversity.md

```diff
@@ -11,3 +11,165 @@ name: tutorial
+)
+
+```
+
+Here we see a significant association between richness and to the bone marrow
+transplant.
+
+We may also be interested in the effect of the auto fecal microbiota
+transplant. It should be known that these are generally correlated, so choosing
+one model over the other will require external knowledge.
+
+
+```{usage}
+use.action(
+    use.UsageAction('longitudinal', 'linear_mixed_effects'),
+    use.UsageInputs(metadata=merged_features,
+                    state_column='day-relative-to-fmt',
+                    individual_id_column='PatientID',
+                    metric='observed_features'),
+    use.UsageOutputNames(visualization='lme-obs-features-FMT')
+)
+```
+
+We also see a downward trend from the FMT. Since the goal of the FMT was to
+ameliorate the impact of the bone marrow transplant, and the timing of the FMT
+is related to the timing of the marrow transplant, we might deduce that the
+negative coefficient is primarily related to the bone marrow transplant and is
+simply echoed in this correlated measure (we can't prove this with statistics
+alone however, instead we are using abductive reasoning).
+
+Looking at the log-likelihood, we also note that the HCT result is slightly
+better than the FMT in accounting for the loss of diversity. But only slightly,
+if we were to perform model testing it may not prove significant.
+
+In any case, we can ask a more targeted question to identify if the FMT was
+useful in recovering diversity.
+
+By adding the ``autoFmtGroup`` to our linear model, we can see if the there is
+are different slopes for the two groups, based on an *interaction term*.
+
+
+```{usage}
+use.action(
+    use.UsageAction('longitudinal', 'linear_mixed_effects'),
+    use.UsageInputs(metadata=merged_features,
+                    state_column='DayRelativeToNearestHCT',
+                    group_columns='autoFmtGroup',
+                    individual_id_column='PatientID',
+                    metric='observed_features'),
+    use.UsageOutputNames(visualization='lme-obs-features-treatmentVScontrol')
+)
+```
+
+Here we see that the ``autoFmtGroup`` is not on its own a significant predictor
+of richness, but its interaction term with ``DayRelativeToNearestHCT`` is.
+This implies that there are different slopes between these groups, and we note
+that given the coding of ``DayRelativeToNearestHCT:autoFmtGroup[T.treatment]``
+we have a positive coefficient which counteracts (to a degree) the negative
+coefficient of ``DayRelativeToNearestHCT``.
+
+```{warning}
+The slopes of the regression scatterplot are not the same fit as our
+mixed-effects model here. They are a naive OLS fit to give a sense of the
+situation. A proper visualization would be a partial regression plot which can
+condition on other terms to show some effect in "isolation". This is not
+currently implemented in QIIME 2.
+```
+
+**How would you test the above models with a different diversity index, such as
```

Comment:
In some of the earlier chapters I posed challenges to create a command, and then hide the answer in the dropdown admonition (see the admonition just above [this heading](https://qiime2-workshops.s3.us-west-2.amazonaws.com/faes-jan2022/030-tutorial-downstream/010-filtering.html#filter-features-from-sequence-data-to-reduce-runtime-of-feature-annotation)). This would fit really well here.